src/app/service/eco-news/create-eco-news.service.ts

```diff
@@ -72,21 +73,21 @@ export class CreateEcoNewsService {
   }
 
   public sendFormData(form, language): Observable<NewsResponseDTO> {
-    const body: NewsDTO = {
-      "imagePath": form.value.source,
+    const body: any = {
       "tags": form.value.tags,
-      "translations": [
-        {
-          "language": {
-            "code": language
-          },
-          "text": form.value.content,
-          "title": form.value.title
-        }
-      ]
+      "text": form.value.content,
+      "title": form.value.title,
     };
+    let formData:FormData = new FormData();
+ 
+    formData.append('image', this.files[0].file, this.files[0].file.name);
+    formData.append('addEcoNewsDtoRequest', JSON.stringify(body));
+
+    console.log(JSON.stringify(body), this.files[0].file, this.files[0].file.name);
 
     this.httpOptions.headers.set('Authorization', `Bearer ${this.accessToken}`);
-    return this.http.post<NewsResponseDTO>(`${this.url}/econews`, body, this.httpOptions);
+    // this.httpOptions.headers.append('Content-Type', 'multipart/form-data;boundary='+Math.random());
+    // this.httpOptions.headers.append('Accept', 'application/json');
```

Comment:
Please, remove the commented code if it is not necessary

Reply:
fixed